x-pack/plugins/security_solution/server/lib/entity_analytics/risk_score/calculate_risk_scores.ts

```diff
@@ -209,8 +209,13 @@ const buildIdentifierTypeAggregation = ({
                 double weighted_score = 0.0;
           
                 fields.put('time', doc['@timestamp'].value);
+                fields.put('rule_name', doc['${ALERT_RULE_NAME}'].value);
```

Comment:
nit: a few bits of whitespace added to this file

Reply:
huh, do you mean I removed the whitespace? 
Wasn't on purpose as far as I can remember, but I think that's actually more inline with our styleguide?

Reply:
Sure, thought it was unintentional